locationHelper.js

```diff
@@ -3,24 +3,29 @@ const path = require("path");
 const preferences = require("./preferencesHelper");
 
 /**
- * @param filepath complete path to .cpp file
+ * Get compiled file
+ * @param language programming language
+ * @param filePath complete path to .c, .cpp or .py file
  */
-function getBinLocation(filepath) {
+function getBinLocation(language, filePath) {
```

Comment:
Maybe we can save the `.pyc` file as `.bin` file too. I don't changing the extension should confuse the Python interpreter. In this way, we can have a common location for all compiled binaries.

Reply:
Actually it works but I don't think it would be the best practice. Also this option is not available in `compileall` or `py_compile` (actually as command line tool). We can move the generated `.pyc` into `binLocation` after it was created and rename it if we want.

Reply:
Oh! In that case, let's leave it as it is.

Reply:
Do you mean it's OK to not to store `.pyc` files in `binLocation`?